lib/internal/bootstrap_node.js

```diff
@@ -257,6 +258,22 @@
     });
   }
 
+  function setupInspectorAPI() {
+    const addCommandLineAPIMethod = process.inspector.addCommandLineAPIMethod;
+    delete process.inspector.addCommandLineAPIMethod;
+    addCommandLineAPIMethod('require', function require(request) {
+      const Module = NativeModule.require('module');
+      const path = NativeModule.require('path');
+      const cwd = tryGetCwd(path);
+
+      const name = '[consoleAPI]';
+      const module = new Module(name);
+      module.filename = path.join(cwd, name);
+      module.paths = Module._nodeModulePaths(cwd);
+      return module.require(request);
+    });
```

Comment:
Shouldn't `module` be defined outside the function?  I think this breaks `require('foo') === require('foo')`, i.e., that the second require returns the cached module instance.